app/client/ui-modules/profiles/state/profile-slice.ts

```diff
@@ -1,103 +1,52 @@
 import { createSlice, PayloadAction } from "@reduxjs/toolkit";
 import { RootState } from "../../../store";
+import { ApiProfileData } from "/app/shared/api-models/user/api-roles/ApiProfileData";
 
-interface ProfileData {
-  firstName: string;
-  lastName: string;
-  dob: string;
-  occupation: string;
-  email: string;
-  phone: string;
-  emergencyContact: string;
-  employer: string;
-  workAddress: string;
-  workPhone: string;
-  profileImage: string;
-  carMake: string;
-  carModel: string;
-  carYear: string;
-  carPlate: string;
-}
-
-interface ProfileState {
-  data: ProfileData;
+const initialState: {
+  data: ApiProfileData;
```

Comment:
you want to use the ProfileData [under domain models] - basically dont use anything api stuff in the client side